apps/charterafrica/src/payload/collections/Documents.js

```diff
@@ -0,0 +1,79 @@
+const Documents = {
```

Comment:
We shouldn't need this, the block should just ask for group name or id... I don't think we'll need multiple groups since they'll setup an account specifically for this.